### DISCUSSION TITLE
How to disable inlay type hints??

### DISCUSSION THREAD

**Discussion by sufferpurrityxd** [microsoft/pylance-release#3129] [Q&A]
![image](https://user-images.githubusercontent.com/102348489/182530505-c08ac706-3edf-4b8c-b502-57138cd9ab88.png)
My settings.json

i turned off 2 "python.analysis.inlayHints.functionReturnTypes" and "python.analysis.inlayHints.variableTypes": false but i still get the type returned

![image](https://user-images.githubusercontent.com/102348489/182530873-af0cf08f-807d-43c4-a84a-c1584269f1c4.png)
![image](https://user-images.githubusercontent.com/102348489/182530910-432df1df-8025-426d-8fe9-6f877509fca9.png)

**Comment by bschnurr** [comment]
`-> str` seems to be actual code.. inlayhints will be grey.

try. `x = 1` and it should become `x: Literal[1] = 1`

turning off inlayhints wont change/edit the actual source code.
